src/System.Private.Xml.Linq/tests/XDocument.Test.ModuleCore/testexception.cs

```diff
@@ -93,8 +93,8 @@ public TestException(TestResult result, string message, object actual, object ex
 
         public override string ToString()
         {
-            var expected = "Expected: " + Expected + " (" + Expected.GetType() + ")\n";
-            var actual = "Actual  : " + Actual + " (" + Actual.GetType() + ")\n";
+            var expected = "Expected: " + Expected + " (" + Expected?.GetType() + ")\n";
```

Comment:
AFAIK null has a ToString() representation and it is represented as an empty string. I did test it locally before fixing the tests when Actual was null and it was fixed.